### DISCUSSION TITLE
I am facing problem in setting up the project!

### DISCUSSION THREAD

**Discussion by kennisjoseph** [coral-xyz/backpack#2384] [General]
![image](https://user-images.githubusercontent.com/74891393/216028869-c3f65499-11e5-4a2f-a7da-37d71c18ee49.png)

Someone help me with this issue, I've followed each steps in the README.md file, Still facing this issue

**Comment by imshubh4141** [comment]
I too am facing issue, after I run `yarn start` simultaneously from both root and packages/app-extension, I get the local host URL. Once the `yarn start` process about to get over from root, it shows an error - run failed

**Comment by Tabish1511** [comment]
Hi,

```yarn install``` runs okay but ```yarn build``` gives me an error at the very end after the line:
```@coral-xyz/app-extension:build: <s> [webpack.Progress] 92% sealing asset processing TerserPlugin```

The error I am getting is below:

```/tmp/xfs-53953e5a/yarn run build exited (1)

 Tasks:    15 successful, 16 total
Cached:    15 cached, 16 total
  Time:    1m36.705s 
Failed:    @coral-xyz/app-extension#build

 ERROR  run failed: command  exited (1)```


Is something wroing with TerserPlugin?
